tstest/integration/integration.go

```diff
@@ -98,3 +110,51 @@ func exe() string {
 	}
 	return ""
 }
+
+// RunDERPAndSTUN runs a local DERP and STUN server for tests, returning the derpMap
+// that clients should use. This creates resources that must be cleaned up with the
+// returned cleanup function.
+func RunDERPAndSTUN(t testing.TB, logf logger.Logf, ipAddress string) (derpMap *tailcfg.DERPMap, cleanup func()) {
```

Comment:
if logf is only going to be t.Logf, you can drop the parameter here and use that directly

Reply:
Some tests use a discard function, so I'm gonna keep it.